bluemix/configuration/core_config/bx_config.go

```diff
@@ -752,9 +754,47 @@ func (c *bxConfig) ClearSession() {
 		c.data.IsLoggedInAsCRI = false
 		c.data.ResourceGroup = models.ResourceGroup{}
 		c.data.LoginAt = time.Time{}
+		c.data.PaginationURLs = []models.PaginationURL{}
 	})
 }
 
+func (c *bxConfig) SetPaginationURLs(paginationURLs []models.PaginationURL) {
+	c.write(func() {
+		c.data.PaginationURLs = paginationURLs
+	})
+}
+
+func (c *bxConfig) ClearPaginationURLs() {
+	c.write(func() {
+		c.data.PaginationURLs = []models.PaginationURL{}
+	})
+}
+
+func (c *bxConfig) AddPaginationURL(index int, url string) error {
+	urls, err := c.PaginationURLs()
+	if err != nil {
+		return err
+	}
+
+	urls = append(urls, models.PaginationURL{
+		LastIndex: index,
+		NextURL:   url,
+	})
+
+	// sort by last index for easier retrieval
+	sort.Sort(models.ByLastIndex(urls))
+	c.SetPaginationURLs(urls)
+	return nil
+}
+
+func (c *bxConfig) PaginationURLs() (paginationURLs []models.PaginationURL, err error) {
```

Comment:
Are you meaning to potentially return an err here? This is different from other getters in the class, and the code doesn't actually return it. Probably just need to remove this from the function signature.

Reply:
Yeah you're right. I think I messed up some commit on a rebase. Will fix